### PR TITLE
2213 webworker connector

### LINKED ISSUE #2213: webworker connector

### PR DESCRIPTION
**Pull request by brianosaurus**
Closes #2213 

Ads a web worker connector and sync to sdk

### CODE REVIEW

File: packages/augur-sdk/src/connector/ww-connector.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { Controller } from "../state/Controller";
+
+const settings = require("@augurproject/sdk/src/state/settings.json");
+
+export class WebWorkerConnector extends Connector {
+  private api: API;
+  private worker: any;
+
+  public async connect(params?: any): Promise<any> {
+    const ethersProvider = new EthersProvider(new JsonRpcProvider(settings.ethNodeURLs[4]), 10, 0, 40);
```
Comment (brianosaurus) [inline packages/augur-sdk/src/connector/ww-connector.ts:23]: All of this stuff is repeated in Sync.ts and here. Could be refactored. Here it exists just to get a db.

File: packages/augur-sdk/src/state/index.ts [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
-    console.log(event.data);
-  };
-} catch (error) {
-  console.log("Your browser does not support web workers");
-}
+    setTimeout(async () => {
+      console.log("Querying");
+      const getMarkets = connector.bindTo(Markets.getMarkets);
+      console.log(await getMarkets({
+        universe: "0x02149d40d255fceac54a3ee3899807b0539bad60",
```
Comment (bthaile) [inline packages/augur-sdk/src/state/index.ts]: Is this just for testing, universe address will change between contract deployments.

Comment (brianosaurus) [reply packages/augur-sdk/src/state/index.ts]: This is just for the test. It should be removed. Doing that now.

**Review by bthaile — APPROVED** [review]
looks better, 👍

**Pull request merged**